src/app/store/slices/user/index.ts

```diff
@@ -83,6 +87,15 @@ export const refreshUserThunk = createAsyncThunk<void, { forceRefresh?: boolean
   },
 );
 
+export const getUserTierFeaturesThunk = createAsyncThunk<void, void, { state: RootState }>(
+  'user/getUserTierFeatures',
+  async (_, { dispatch }) => {
+    const userFeatures = await getAvailableUserFeatures();
+
+    dispatch(userActions.setUserTierFeatures(userFeatures));
```

Comment:
There is no case handled if this fails or takes too long, perhaps we should implement a warning that there may be some unused features due to the failure 🤔